py/msedge/selenium_tools/webdriver.py

```diff
@@ -31,7 +31,7 @@ class WebDriver(RemoteWebDriver):
 
     def __init__(self, executable_path='',
                  capabilities=None, port=0, verbose=False, service_log_path=None,
-                 log_path=None, keep_alive=False,
+                 log_path=None, keep_alive=True,
```

Comment:
Legacy Edge still expects keep_alive=False by default.

Please change keep_alive to be None by default. If the value is None (in other words, user didn't provide a value) then set it to True or False depending on use_chromium.

This will preserve the default behavior for both legacy and chromium Edge.